packages/core/src/Components/markers/marker-line.jsx

```diff
@@ -13,7 +13,7 @@ const MarkerLine = ({ label, line_style, marker_config, status }) => {
                 <Icon
                     icon='IcContractExitTimeCircle'
                     className='chart-marker-line__icon'
-                    color={status === 'lost' ? 'red' : 'green'}
+                    custom_color={status === 'lost' ? '#e6190e' : '#008832'}
```

Comment:
Can't we use the ones from `shared` package?

Reply:
You mean BARRIER_COLORS? From packages/shared/src/utils/constants/barriers.ts?
Yes, I think we can